src/BuiltInTools/dotnet-watch/dotnet-watch.csproj

```diff
@@ -43,39 +43,17 @@
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\BrowserRefresh\Microsoft.AspNetCore.Watch.BrowserRefresh.csproj"
```

Comment:
Was the file formatting done on purpose? One attribute per line makes csprojs easier to read.

Reply:
Not on purpose, VS did it automatically :( Will undo.